app/theme/server/server.js

```diff
@@ -80,7 +81,8 @@ export const theme = new class {
 
 			currentHash = crypto.createHash('sha1').update(data.css).digest('hex');
 			currentSize = data.css.length;
-			Inject.rawHead('css-theme', `<link rel="stylesheet" type="text/css" href="/theme.css?${ currentHash }">`);
+			const themePath = `/theme.css?${ currentHash }`;
+			Inject.rawHead('css-theme', `<link rel="stylesheet" type="text/css" href=${ getURL(themePath) }>`);
```

Comment:
doesn't this still need to be in `"` ?

Reply:
Does not need, does not change behavior, but I'll add to keep the pattern.